src/inject/dynamic-theme/index.ts

```diff
@@ -130,6 +130,14 @@ function createStaticStyleOverrides() {
     const proxyScript = createOrUpdateScript('darkreader--proxy');
     proxyScript.textContent = `(${injectProxy})()`;
     document.head.insertBefore(proxyScript, variableStyle.nextSibling);
+
+    const dynamicVariableStyle = createOrUpdateStyle('darkreader--dynamicVariable');
```

Comment:
Let's follow our CSS selectors convention and use spinal case `darkreader--dynamic-variables`